tidb-cloud/data-service-manage-endpoint.md

```diff
@@ -120,6 +120,15 @@ On the right pane of the endpoint details page, you can click the **Properties**
 #### Advanced properties
 
 - **Tag**: the tag used for identifying a group of endpoints.
+- **Pagination**: this property is available only when the request method is `GET` and the last SQL statement of the endpoint is a `SELECT` operation. When **Pagination** is enabled, you can paginate the results by specifying `page` and `page_size` as query parameters when calling the endpoint, such as `https://<region>.data.tidbcloud.com/api/v1beta/app/<App ID>/endpoint/my_endpoint/get_id?page=<Page Number>&page_size=<Page Size>`. For more information, see [Call an endpoint](#call-an-endpoint).
+
+    > **Note:**
+    >
+    > - If you do not include the `page` and `page_size` parameters in the request, the default behavior is to return the maximum number of rows specified in the **Max Rows** property on a single page.
+    > - The `page_size` must be less than or equal to the **Max Rows** property. Otherwise, an error is returned.
+
+- **Cache Response**: this property is available only when the request method is `GET`. When **Cache Response** is enabled, TiDB Cloud Data Service can cache the response returned by your `GET` requests within a specified time-to-live (TTL) period.
+- **Time-to-live**: this property is available only when **Cache Response** is enabled. You can use it to specify the time-to-live (TTL) period of cached response. During the TTL period, if you make the same `GET` requests again, Data Service returns the cached response directly instead of fetching data from the target database again, which improves your query performance.
```

Comment:
```suggestion
- **Time-to-live**: this property is available only when **Cache Response** is enabled. You can use it to specify the time-to-live (TTL) period in seconds for cached response. During the TTL period, if you make the same `GET` requests again, Data Service returns the cached response directly instead of fetching data from the target database again, which improves your query performance.
```